sql/expression_value.h

```diff
@@ -807,6 +807,31 @@ struct ExpressionValue {
 
     Type type_;
 
+    static inline std::string print(Type t) {
+        switch (t)  {
+        case NONE:      return "empty";
+        case ATOM:      return "atomic value";
```

Comment:
Although _atom_ is still present in the documentation, I thought we were using _scalar_ now.  @nicolaskruchten ?